help/develop/admin/deploy/deploy.md

```diff
@@ -105,11 +109,11 @@ as [AWS EKS](https://aws.amazon.com/eks/), you can use our prepared deployment s
 ### Regular machine deployment
 
 Datagrok can be deployed to a regular machine: bare-metal servers or virtual machines. This method is not as reliable,
-scalable, and maintainable as others, so we do not recommend it for production usage. This deployment method will
+scalable, and maintainable as others, so **we do not recommend it for production usage**. This deployment method will
 require separate deployment for Datagrok required resources: PostgreSQL database and persistent storage(local filesystem
 can be used).
 
-We use native Docker compose commands to run applications on machines. It simplifies multi-container application
+We use native Docker compose commands to run platform components on machines. It simplifies multi-container application
 development and deployment.
 
 More information:
```

Comment:
There should be "Next steps" with configuration for smtp and auth, which are important part of platform first-time deployment